spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "spec_helper"
+
+describe UsersController do
+
+  describe 'GET #new' do
+    before do
+      get :new
+    end
+
+    it 'should render the new template' do
+      expect(response).to render_template :new
+    end
+
+    it 'should sets the @user variable' do
+      expect(assigns(:user)).to be_a_new(User)
+    end
+  end
+
+  describe 'POST #create' do
+
+    context "with valid input" do
+      before do
+        post :create, user: Fabricate.attributes_for(:user)
+      end
+
+      it 'should create the user' do
+        expect(assigns(:user)).not_to be_a_new(User)
```

Comment:
This will pass as long as it is ANYTHING but a new `User`. I would recommend testing `be_persisted`:

```ruby
expect(assigns(:user)).to be_persisted
```